cernopendata/modules/fixtures/data/records/cms-validation-code-jet-tuple.json

```diff
@@ -0,0 +1,72 @@
+[
+{
+  "abstract": {
+       "description": " <p>This code  is a CMSSW module producing flat tuples from 2011A Jet data, set up at Research level, i.e. it requires university-student-level programming experience. Minimal acquaintance with Linux and the ROOT analysis package (<a href=\"https://root.cern.ch/\">https://root.cern.ch/</a>) as well as a basic text editor is needed. </p>"
+   },
+   "accelerator": "CERN-LHC", 
+   "authors": [
+     {
+       "name": "Zenaiev, Oleksandr"
+     },
+     {
+       "name": "Haapalehto, Matias"
+     }
+   ], 
+   "collections": [
+     "CMS-Validation-Utilities"
+   ],
+   "date_created": "2011", 
+   "date_published": "2017", 
+   "distribution": {
+    "formats": [
+        "gz"
+    ],
+    "number_files": 1, 
+    "size": 16966
+  },
+  "doi": "10.7483/OPENDATA.CMS.JT93.12XI",
+  "experiment": "CMS",
+    "files": [
+    {
+      "checksum": "sha1:1361f245ac6f9e909ff530c75a5db0e2ef84a348",
+      "size": 16966,
+      "uri": "root://eospublic.cern.ch//eos/opendata/cms/software/2011-jet-inclusivecrosssection-ntupleproduction-optimized/2011-jet-inclusivecrosssection-ntupleproduction-optimized-1.0.0.tar.gz"
+    }
+  ],
+  "license": {
+    "attribution": "GNU General Public License (GPL) version 3"
```

Comment:
@ArtemisLav @katilp BTW we should check the licenses later, there are often none on GitHub...